src/modules/data/PublicKey.ts

```diff
@@ -37,17 +37,32 @@ export class PublicKey
     public static Width: number = sodium.crypto_sign_PUBLICKEYBYTES;
 
     /**
-     * Constructor
-     * @param bin The binary data of the public key
+     * Construct a new instance of this class
+     *
+     * @param data   - The string or binary representation of the public key
+     * @param endian - The byte order
```

Comment:
JSDoc comment also supports `-`.
Reading is good if `-` are included.
However, most documents do not have `-`.
https://jsdoc.app/tags-param.html

Reply:
Indeed, will revert